lib/OfxParser/Ofx.php

```diff
@@ -200,14 +223,157 @@ private function buildCreditAccount(SimpleXMLElement $xml)
 
         $creditAccount->statement = new Statement();
         $creditAccount->statement->currency = $xml->CCSTMTRS->CURDEF;
-        $creditAccount->statement->startDate = $this->createDateTimeFromStr($xml->CCSTMTRS->BANKTRANLIST->DTSTART);
-        $creditAccount->statement->endDate = $this->createDateTimeFromStr($xml->CCSTMTRS->BANKTRANLIST->DTEND);
+        $creditAccount->statement->startDate = $this->createDateTimeFromStr($xml->CCSTMTRS->BANKTRANLIST->DTSTART, true);
+        $creditAccount->statement->endDate = $this->createDateTimeFromStr($xml->CCSTMTRS->BANKTRANLIST->DTEND, true);
         $creditAccount->statement->transactions = $this->buildTransactions($xml->CCSTMTRS->BANKTRANLIST->STMTTRN);
 
         return $creditAccount;
     }
 
     /**
+     * @param SimpleXMLElement $xml
+     * @return BankAccount
+     * @throws \Exception
+     */
+    private function buildInvestmentAccount(SimpleXMLElement $xml)
+    {
+        $nodeName = 'INVACCTFROM';
+        if (!isset($xml->INVSTMTRS->$nodeName)) {
+            $nodeName = 'BANKACCTFROM';
```

Comment:
Do we really want this to fall back to `BANKACCTFROM` ? Surely that is handled by `buildBankAccount` etc.?